module2/homework/generateSequence/generateSequence.hpp

```diff
@@ -2,6 +2,12 @@
 #include <vector>
 
 std::vector<int> generateSequence(int count, int step) {
-    // TODO: Implement me :)
-    return {};
-}
+    if (count<=0) {return {};}
+    else {
+        std::vector<int> vec {step};
+        for (int i=1; i<count; i++){
```

Comment:
```suggestion
        for (int i = 1; i < count; i++){
```
Using spaces next to operators improve readability